elastic/integrations
# Update ECS version 8.0.0 to 8.5.1

- Enhancement

## What does this PR do?

Updates ECS version from 8.0.x to 8.5.1

## Checklist


## How to test this PR locally

Verify the ECS version is update in build.yml and in ingest pipeline. Execute system and pipeline tests.

- Closes #4788 
- Closes #5067
